common/lib/dependabot/utils.rb

```diff
@@ -54,7 +59,21 @@ def self.always_clone_for_package_manager?(package_manager)
 
     sig { params(package_manager: String).void }
     def self.register_always_clone(package_manager)
+      validate_package_manager!(package_manager)
+
       @cloning_package_managers << package_manager
     end
+
+    sig { params(package_manager: String).void }
+    def self.validate_package_manager!(package_manager)
+      # Official package manager
+      return if Config::File::PACKAGE_MANAGER_LOOKUP.invert.key?(package_manager)
```

Comment:
The Sorbet error is because this isn't typed:

https://github.com/dependabot/dependabot-core/blob/ba47d42101b843867e05f4c24d6e45e15b1a3076/common/lib/dependabot/config/file.rb#L49-L67

The fix is to wrap it in `T.let(..., T::Hash[String, String])`

Reply:
Thank you!

Reply:
You're welcome! But I should have been more explicit. It should be:

```rb
PACKAGE_MANAGER_LOOKUP = T.let({
        "bundler" => "bundler",
        "cargo" => "cargo",
        "composer" => "composer",
        "docker" => "docker",
        "elm" => "elm",
        "github-actions" => "github_actions",
        "gitsubmodule" => "submodules",
        "gomod" => "go_modules",
        "gradle" => "gradle",
        "maven" => "maven",
        "mix" => "hex",
        "nuget" => "nuget",
        "npm" => "npm_and_yarn",
        "pip" => "pip",
        "pub" => "pub",
        "swift" => "swift",
        "terraform" => "terraform"
      }.freeze, T::Hash[String, String])
```